src/frontend/screens/ObservationCreate/index.tsx

```diff
@@ -88,13 +88,27 @@ export const ObservationCreate = ({
   navigation,
 }: NativeRootNavigationProps<'ObservationCreate'>) => {
   const {formatMessage} = useIntl();
+  const {projectId} = useActiveProject();
   const {usePreset} = useDraftObservation();
   const preset = usePreset();
   const value = usePersistedDraftObservation(store => store.value);
   const attachments = usePersistedDraftObservation(store => store.attachments);
   const {updateTags, clearDraft} = useDraftObservation();
-  const createObservationMutation = useCreateObservation();
-  const createBlobMutation = useCreateBlobMutation();
+  const {
+    mutate: createObservationMutation,
+    reset: resetObservation,
+    status: observationStatus,
+    error: observationError,
+  } = useCreateDocument({
+    docType: 'observation',
+    projectId,
+  });
+  const {
+    mutateAsync: createBlobAsync,
```

Comment:
update name to be more accurate:

```suggestion
    mutateAsync: createAttachmentAsync,
```